dev/io.openliberty.restfulWS30.appSecurity/bnd.bnd

```diff
@@ -38,10 +38,10 @@ Import-Package: \
  jakarta.ws.rs.container, \
  jakarta.ws.rs.core, \
  jakarta.ws.rs.ext, \
- org.jboss.resteasy.annotations.cache, \
- org.jboss.resteasy.plugins.interceptors, \
- org.jboss.resteasy.spi, \
- org.jboss.resteasy.util, \
+ org.jboss.resteasy.annotations.cache;version="[4.7,6.2.0)", \
```

Comment:
yeah you already updated the import so no need for the separate feature and bundle.

Reply:
agreed